modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1654,22 +1652,12 @@ definition {
 			ObjectAdmin.goToCustomObject(objectName = "CustomObject162179");
 
 			ObjectPortlet.addCustomFieldEntryViaUI(
-				customField = "Custom Integer",
-				entry = "123");
+				customField = "Custom Field",
+				entry = "Entry Test");
 
 			AssertTextEquals.assertPartialText(
 				locator1 = "Message#ERROR",
 				value1 = "This entry is not possible.");
-
-			Navigator.gotoBack();
-
-			ObjectPortlet.addCustomFieldEntryViaUI(
-				customField = "Custom Integer",
-				entry = "2147483647");
-
-			ObjectAdmin.goToCustomObject(objectName = "CustomObject162179");
-
-			ObjectPortlet.viewEntry(entry = "2147483647");
```

Comment:
Hey @evanilson-santana,

Add a correct entry with a string field. You need to confirm the correct entry for more precision on testcase.